### PR TITLE
added short intros to section landing pages

### PR DESCRIPTION
**Pull request by diana-pilvar**

### CODE REVIEW

File: data-sources/index.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 title: Data sources
 ---
 
+Identifying and sharing data from related data sources is important for comprehensive analysis and informed decision-making. Data sources may include clinical data from hospitals, laboratory results, epidemiological surveys, and public health records.Integrating these varied data sources enable researchers and health professionals to track disease spread, evaluate the effectiveness of interventions, and predict future outbreaks. Sharing data among various stakeholders, while maintaining privacy and ethical standards, supports real-time monitoring, and aids in the global effort to control infectious diseases.
```
Comment (hedi-ee) [inline data-sources/index.md:5]:
```suggestion
Identifying and sharing data from related data sources is important for comprehensive analysis and informed decision-making. Data sources may include clinical data from hospitals, laboratory results, epidemiological surveys, and public health records. Integrating these varied data sources enables researchers and health professionals to track disease spread, evaluate the effectiveness of interventions, and predict future outbreaks. Sharing data among various stakeholders while maintaining privacy and ethical standards supports real-time monitoring and aids in the global effort to control infectious diseases.
```

File: quality-control/index.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 title: Quality control
 ---
 
+Quality control ensures the accuracy and reliability of data collected and analyzed. Various tools and methods are used at different stages—from collection and processing to analysis and reporting. Effective quality control measures help identify and correct errors, reduce data variability, and enhance the credibility of findings. By implementing rigorous quality control protocols, researchers can provide robust, actionable insights that are critical for managing and mitigating infectious disease outbreaks.
```
Comment (hedi-ee) [inline quality-control/index.md:5]:
```suggestion
Quality control ensures the accuracy and reliability of data collected and analysed. Various tools and methods are used at different stages—from collection and processing to analysis and reporting. Effective quality control measures help identify and correct errors, reduce data variability, and enhance the credibility of findings. By implementing rigorous quality control protocols, researchers can provide robust, actionable insights that are critical for managing and mitigating infectious disease outbreaks.
```

File: tools-and-resources-list/index.md [hunk 3]
```diff
@@ -3,6 +3,7 @@ title: Tools and resources list
 datatable: true
 ---
 
+ To effectively address the complexities of infectious diseases, professionals can browse all tools and resources mentioned in an infectious diseases toolkit. This approach ensures that stakeholders are well-equipped with reliable and proven tools to combat infectious diseases.
```
Comment (hedi-ee) [inline tools-and-resources-list/index.md:6]:
```suggestion
This page captures all tools and resources mentioned across the Infectious Diseases Toolkit. 
```

File: data-description/index.md [hunk 4]
```diff
@@ -2,6 +2,7 @@
 title: Data description
 ---
 
+Data description involves detailing the datasets used in research or surveillance. This includes finding and adhering to relevant (meta)data standards and documentation practices to ensure data are accurately represented, interoperable, and reusable. Accurate data description improves the clarity and dependability of information regarding disease incidence, transmission, and treatment outcomes, which is necessary for effective public health interventions and scientific studies. It also promotes data sharing among various stakeholders, allowing for more metaanalyses and encouraging collaborations that can improve disease understanding and management.
```
Comment (hedi-ee) [inline data-description/index.md:5]:
```suggestion
Data description involves detailing the datasets used in research or surveillance. This, for example, includes finding and adhering to relevant (meta)data standards and documentation practices to ensure data are accurately represented, interoperable, and reusable. Accurate data description improves the clarity and dependability of information regarding disease incidence, transmission, and treatment outcomes, which is necessary for effective public health interventions and scientific studies. Good data description promotes data sharing among various stakeholders, allowing for more meta-analyses and encouraging collaborations to improve disease understanding and management.
```

File: data-analysis/index.md [hunk 5]
```diff
@@ -2,6 +2,7 @@
 title: Data analysis
 ---
 
+Data analysis involves employing generic workflows tailored to different types of data, such as epidemiological data, clinical data, or molecular data. These workflows include steps like data collection, cleaning, integration, and analysis, using statistical and computational methods to uncover trends, associations, and patterns. This systematic approach helps researchers and health professionals understand disease dynamics, assess intervention strategies, and improve disease prevention and control measures based on robust, data-driven insights.
```
Comment (hedi-ee) [inline data-analysis/index.md:5]:
```suggestion
Data analysis involves employing workflows tailored to different types of data, such as epidemiological, clinical, or molecular data. These workflows can include steps like data collection, cleaning, integration, and analysis, using statistical and computational methods to uncover trends, associations, and patterns. Systematic data analysis helps researchers and health professionals understand disease dynamics, assess intervention strategies, and improve disease prevention and control measures based on robust, data-driven insights.
```

File: showcase/index.md [hunk 6]
```diff
@@ -2,6 +2,8 @@
 title: Showcase
 ---
 
+Showcasing involves presenting applied examples of handling infectious disease data to demonstrate successful strategies and methodologies. Showcasing these examples not only highlights best practices but also fosters learning and adoption of proven techniques across the public health and scientific communities. This is crucial for building a robust response to infectious diseases globally, as it encourages the sharing of knowledge and experiences that can lead to improved health outcomes.
```
Comment (hedi-ee) [inline showcase/index.md:5]:
```suggestion
Showcasing involves presenting applied examples of handling infectious disease data to demonstrate successful strategies and methodologies across different countries and domains. Showcases highlight best practices and foster learning and adoption of proven techniques across the public health and scientific communities. 
```

**Review by hedi-ee — COMMENTED** [review]
Adjusted the intros

**Review by bedroesb — APPROVED** [review]

File: data-analysis/index.md [hunk 7]
```diff
@@ -2,6 +2,7 @@
 title: Data analysis
 ---
 
+Data analysis involves employing workflows tailored to different types of data, such as epidemiological, clinical, or molecular data. These workflows can include steps like data collection, cleaning, integration, and analysis, using statistical and computational methods to uncover trends, associations, and patterns. Systematic data analysis helps researchers and health professionals understand disease dynamics, assess intervention strategies, and improve disease prevention and control measures based on robust, data-driven insights.
```
Comment (LianeHughes) [inline data-analysis/index.md:5]:
```suggestion
Data analysis involves employing workflows tailored to different types of data, such as epidemiological, clinical, or molecular data. These workflows can include steps like data collection, cleaning, integration, and analysis, and use statistical and computational methods to uncover trends, associations, and patterns. Systematic data analysis helps researchers and health professionals to understand disease dynamics, assess intervention strategies, and improve disease prevention and control measures based on robust, data-driven insights.
```

File: data-description/index.md [hunk 8]
```diff
@@ -2,6 +2,7 @@
 title: Data description
 ---
 
+Data description involves detailing the datasets used in research or surveillance. This, for example, includes finding and adhering to relevant (meta)data standards and documentation practices to ensure data are accurately represented, interoperable, and reusable. Accurate data description improves the clarity and dependability of information regarding disease incidence, transmission, and treatment outcomes, which is necessary for effective public health interventions and scientific studies. Good data description promotes data sharing among various stakeholders, allowing for more meta-analyses and encouraging collaborations to improve disease understanding and management.
```
Comment (LianeHughes) [inline data-description/index.md:5]:
```suggestion
Data description involves detailing the datasets used in research or surveillance. This includes, for example, finding and adhering to relevant standards for (meta)data and documentation to ensure that  data are accurately represented, interoperable, and reusable. Accurate data description improves the clarity and reliability of information regarding disease incidence, transmission, and treatment outcomes, which is essential for ensuring effective public health interventions and scientific studies. Good data description promotes data sharing among various stakeholders, allowing for more meta-analyses, and encouraging collaborations to improve the understanding and management of disease.
```

File: data-sources/index.md [hunk 9]
```diff
@@ -2,6 +2,7 @@
 title: Data sources
 ---
 
+Identifying and sharing data from related data sources is important for comprehensive analysis and informed decision-making. Data sources may include clinical data from hospitals, laboratory results, epidemiological surveys, and public health records. Integrating these varied data sources enables researchers and health professionals to track disease spread, evaluate the effectiveness of interventions, and predict future outbreaks. Sharing data among various stakeholders while maintaining privacy and ethical standards supports real-time monitoring and aids in the global effort to control infectious diseases.
```
Comment (LianeHughes) [inline data-sources/index.md:5]:
```suggestion
Identifying and sharing data from related sources is important for comprehensive analysis and informed decision-making. Data sources may include clinical data from hospitals, laboratory results, epidemiological surveys, and public health records. Integrating these varied data sources enables researchers and health professionals to track disease spread, evaluate the effectiveness of interventions, and predict future outbreaks. Sharing data among various stakeholders whilst maintaining privacy and ethical standards supports real-time monitoring and aids in the global effort to control infectious diseases.
```

File: provenance/index.md [hunk 10]
```diff
@@ -2,6 +2,7 @@
 title: Provenance
 ---
 
+Provenance refers to the practice of meticulously tracking data and analysis steps to ensure the integrity and reproducibility of research findings. This involves documenting the origins of data, all transformations and manipulations it undergoes, and the analytical methods applied. By maintaining detailed records of these elements, researchers can provide a transparent path from raw data to final results, facilitating validation and verification of the data by other researchers. This is critical in infectious disease research where data accuracy and reliability are essential for effective disease monitoring, outbreak response, and public health decision-making.
```
Comment (LianeHughes) [inline provenance/index.md:5]:
```suggestion
Provenance refers to the practice of meticulously tracking data, as well as associated processing and analysis steps, to ensure the integrity and reproducibility of research findings. This involves documenting the origins of data, all transformations and manipulations it undergoes, and the analytical methods applied. By maintaining detailed records of these elements, researchers can provide a transparent path from raw data to final results, facilitating the validation and verification of the data by other researchers. This is critical in infectious disease research, where data accuracy and reliability are essential for effective disease monitoring, outbreak response, and public health decision-making.
```

File: quality-control/index.md [hunk 11]
```diff
@@ -2,6 +2,7 @@
 title: Quality control
 ---
 
+Quality control ensures the accuracy and reliability of data collected and analysed. Various tools and methods are used at different stages—from collection and processing to analysis and reporting. Effective quality control measures help identify and correct errors, reduce data variability, and enhance the credibility of findings. By implementing rigorous quality control protocols, researchers can provide robust, actionable insights that are critical for managing and mitigating infectious disease outbreaks.
```
Comment (LianeHughes) [inline quality-control/index.md:5]:
```suggestion
Quality control ensures the accuracy and reliability of data collected and analysed. Various tools and methods are used at different stages, from collection and processing to analysis and reporting. Effective quality control measures help to identify and correct errors, reduce data variability, and enhance the credibility of findings. By implementing rigorous quality control protocols, researchers can provide robust, actionable insights that are critical for managing and mitigating infectious disease outbreaks.
```

File: showcase/index.md [hunk 12]
```diff
@@ -2,6 +2,8 @@
 title: Showcase
 ---
 
+Showcasing involves presenting applied examples of handling infectious disease data to demonstrate successful strategies and methodologies across different countries and domains. Showcases highlight best practices and foster learning and adoption of proven techniques across the public health and scientific communities.
```
Comment (LianeHughes) [inline showcase/index.md:5]:
```suggestion
Showcasing involves presenting applied examples of handling infectious disease data to demonstrate successful strategies and methodologies that have been developed and used across different countries and domains. Showcases highlight best practices, encourage learning, and facilitate the adoption of proven techniques across both the public health and scientific communities.
```

**Review by LianeHughes — APPROVED** [review]

**Pull request merged**